### ISSUE TITLE
Test the feasibility if running SCT manager tests triggered from siren-tests framework

### ISSUE THREAD

**Issue by mikliapko** [scylladb/scylla-manager#4118]
Some time ago, some attempts to "marry" siren-tests and SCT have been already done which resulted in siren longevity tests where the cluster is created in Cloud and subjected to SCT nemesises.

Currently, these tests are disabled - not clear in which state they are and how much effort is required to bring them back to life.

The scope of this task is:
- at first, bring back to live existing tests and run any of them;
- run any Manager SCT test (preferable for Manager restore operation) from siren.

**Comment by mikliapko** [comment]
Temporarily blocked by SCT issue https://github.com/scylladb/scylla-cluster-tests/issues/9427

**Comment by mikliapko** [comment]
Job with all the test attempts:
https://jenkins.scylladb.com/job/scylla-staging/job/mikita/job/siren-tests/job/siren-sct-integration/

**Comment by mikliapko** [comment]
[Job](https://jenkins.scylladb.com/job/scylla-staging/job/mikita/job/siren-tests/job/siren-sct-integration/93/) confirms that SCT setup passes successfully with the cluster created in Cloud.

Consider it as a PoC for the sct+siren-tests testing approach.

**Comment by mikliapko** [comment]
PoC run: https://jenkins.scylladb.com/job/scylla-staging/job/mikita/job/siren-tests/job/siren-sct-integration/109/
